lib/core/src/Cardano/Wallet/Primitive/Types/Tx.hs

```diff
@@ -7,6 +7,7 @@
 {-# LANGUAGE GeneralizedNewtypeDeriving #-}
 {-# LANGUAGE NamedFieldPuns #-}
 {-# LANGUAGE OverloadedLabels #-}
+{-# LANGUAGE RankNTypes #-}
```

Comment:
Whoops. Added for implementing a function I didn't end up using. This got included by mistake.